caesar.c

```diff
@@ -0,0 +1,120 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include "caesar.h"
+
+//Function Declarations
+int _CheckChar(char chr);
+int _CheckNumOfShifts(int num, int* ALPshift, int* DGshift);
+void _EncodeChar(char* add_char, int shiftDirect, int modValue, int numShift, int arraySize);
+
+//Function Definitions 
+Input GetInputStringInfo(char* string)
+{
+	Input stringInfo;
+	//Get number of shifts
+	stringInfo.numShift = atoi(strtok(string, ":"));
```

Comment:
what is string is null? Also, avoid choosing a variable name that is commonly a reserved word in most languages